src/drivers/distance_sensor/vl53l1x/vl53l1x.cpp

```diff
@@ -166,16 +176,16 @@ VL53L1X::~VL53L1X()
 int VL53L1X::collect()
 {
 	uint8_t ret = 0;
-	uint8_t rangeStatus;
+	uint8_t rangeStatus = 0;
 	uint16_t distance_mm = 0;
 
 	perf_begin(_sample_perf);
 
 	const hrt_abstime timestamp_sample = hrt_absolute_time();
 
 	ret = VL53L1X_GetRangeStatus(&rangeStatus);
-
-	if ((ret != PX4_OK) | (rangeStatus != PX4_OK)) {
+	
+	if ((ret != PX4_OK) | (rangeStatus == 13)) {
```

Comment:
Is there a range status define you can use instead of the magic number (13)?

Reply:
this is a range status that indicates ROI is 'out of bounds', also referenced in the ST multiple zone manaul

Reply:
Please create const values for each of those, e.g. RANGE_STATUS_OUT_OF_BOUNDS for this one and then use it.